sdk/core/azure_core/src/http/policies/instrumentation/request_instrumentation.rs

```diff
@@ -43,16 +43,16 @@ impl RequestInstrumentationPolicy {
     ///
     pub fn new(
         tracer: Option<Arc<dyn crate::tracing::Tracer>>,
-        log_options: &LoggingOptions,
+        logging_options: &LoggingOptions,
```

Comment:
I wouldn't hold up this PR for it, but if you think there's enough reason to specify these as "logging options" - that there might be more options in the future - than I suggest making a single `RequestInstrumentationPolicyOptions` that contains `logging` ("options" is then redundant) to better avoid breaking changes in the future.